tests/integration.rs

```diff
@@ -5565,6 +5565,32 @@ mod tests {
             }
         }
 
+        fn get_cpu_count_windows() -> u8 {
+            let auth = windows_auth();
```

Comment:
I think we can assume the authentication is already done. Make sure the authentication is performed from the test itself.

Reply:
But it's just creating a struct to be passed to the ssh command, it doesn't perform any auth. Can sure pass an existing object here from the upper level.

Thanks.

Reply:
Fixed, passing the auth object reference now.

Thanks